assets/js/theme/common/product-details.js

```diff
@@ -376,6 +376,10 @@ export default class ProductDetails extends ProductDetailsBase {
             if (this.previewModal) {
                 this.previewModal.open();
 
+                if (window.ApplePaySession) {
+                    this.previewModal.getJqueryElement().addClass('apple-pay-supported');
+                }
```

Comment:
Can we get access to the element like this.previewModal.$modal&?

Reply:
Updated